src/testsuite/vec_pwr9_dummy.c

```diff
@@ -1735,9 +1745,13 @@ __test_madd512x512a512_PWR9 (__VEC_U_512 m1, __VEC_U_512 m2, __VEC_U_512 a1)
 {
   __VEC_U_1024 result;
   vui128_t mc, mp, mq;
+#ifndef __clang__
   register vui128_t t0 asm("vs10");
   register vui128_t t1 asm("vs11");
   register vui128_t t2 asm("vs12");
+#else
+  vui128_t t0, t1, t2;
+#endif
```

Comment:
why? add a comment.

Reply:
// Clang does not support register variables.